dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -188,3 +188,6 @@ excludeAllHandledTypesClasses.desc=When this property is set to true, during sta
 
 skipEncodedCharVerification.name=Skip verification of encoded characters in URI
 skipEncodedCharVerification.desc=A request is rejected if the request URI contains the %23, %2E, %2F, or %5C encoded characters. When this property is set to true, the request URI is not checked for these encoded characters. This property is available starting in servlet 6.0, and default value is false.
+
+donotCloseOutputOnForwardForServletError.name=Do not close output on forwarded servlet error 
+donotCloseOutputOnForwardForServletError.desc=When this property is set to true, if servlet called from dispatch forward throws exceptions other than IOException and ServletException they will not have output automatically closed.
```

Comment:
We should have ID review the description

Reply:
done